float64 l_scale
float64 l_offs_x
float64 l_offs_y
float64 l_offs_z
float64 r_scale
float64 r_offs_x
float64 r_offs_y
float64 r_offs_z